### ISSUE TITLE
Focus is not trapped inside Dialog

### ISSUE THREAD

**Issue by TGlide** [WailAbou/shadcn-svelte-nodep#11]
According to the [WAI-ARIA Dialog Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/), pressing tab should only focus items inside the dialog.

**Comment by WailAbou** [comment]
Thank you, it should be fixed with this [PR](https://github.com/WailAbou/shadcn-svelte-nodep/pull/37) along with focus trap for the alert dialog and popover components.
